specs/core/0_beacon-chain.md

```diff
@@ -1677,8 +1677,9 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
     if pubkey not in validator_pubkeys:
         # Verify the deposit signature (proof of possession) for new validators.
         # Note: The deposit contract does not check signatures.
-        # Note: Deposits are valid across forks, hence the deposit domain is retrieved directly from `bls_domain`
-        if not bls_verify(pubkey, signing_root(deposit.data), deposit.data.signature, bls_domain(DOMAIN_DEPOSIT)):
+        # Note: Deposits are valid across forks. thus the deposit domain is retrieved directly from `compute_bls_domain`
+        if not bls_verify(pubkey, signing_root(deposit.data),
+                          deposit.data.signature, compute_bls_domain(DOMAIN_DEPOSIT)):
```

Comment:
add `domain` variable and split into two

Reply:
Will do